nucliadb/nucliadb/search/api/v1/search.py

```diff
@@ -221,8 +220,17 @@ async def search(
                 ),
             )
     else:
-        if item.sort is not None and item.sort.limit is None:
-            item.sort.limit = SORTED_RELEVANT_SEARCH_LIMIT
+        if item.sort is None:
```

Comment:
NIT: since the logic for validating the sort options is getting complex, I would move it into a separate function.

Reply:
Done